src/include/splash/core/DCDataSet.hpp

```diff
@@ -34,6 +34,8 @@
 #include "splash/CollectionType.hpp"
 #include "splash/basetypes/ColTypeDim.hpp"
 
+herr_t H5Ewalk_strings(unsigned n, const H5E_error2_t *err_desc, void *client_data);
```

Comment:
Full ack on this. This basically means, since we have to go through all the exceptions anyway, we will use this pull request to also bundle the (seven) different `getExceptionString` implementations in one common helper.